support-models/src/main/scala/com/gu/support/workers/PaymentProvider.scala

```diff
@@ -43,17 +43,16 @@ object PaymentProvider {
 
   implicit val encoder: Encoder[PaymentProvider] = Encoder.encodeString.contramap[PaymentProvider](_.toString)
 
-  def fromPaymentFields(paymentFields: Option[PaymentFields]): PaymentProvider = paymentFields match {
```

Comment:
In the past this could have been None for gift redemptions, now it will always be present